src/logic/map_objects/tribes/production_program.cc

```diff
@@ -1365,7 +1351,7 @@ then leave the site looking for employment. The produced workers are of the type
          "return=skipped unless economy needs atlanteans_horse",
          "consume=corn water",
          "sleep=duration:15s",
-         "playsound=sound/farm/horse priority:50% allow_multiple",
+         "playsound=sound/farm/horse priority:50, allow_multiple",
```

Comment:
```suggestion
         "playsound=sound/farm/horse priority:50% allow_multiple",
```

sorry, forgot that I wanted to convert these to suggestions now that I figured them out...